File: api/Entities/Admin/Order/OrderForwardCategory.cs
    namespace api.Entities.Admin.Order
{
    public class OrderForwardCategory: BaseEntity
    {
        public int OrderId { get; set; }
        public int OrderNo {get; set; }
        public DateTime OrderDate { get; set; }
        public string CustomerName { get; set; }
        public string CustomerCity { get; set; }
        public int OrderItemId { get; set; }
        //public int OrderForwardToAgentId { get; set; }
        public int ProfessionId { get; set; }
        public string ProfessionName { get; set; }
        public int Charges { get; set; }
        public ICollection<OrderForwardCategoryOfficial> OrderForwardCategoryOfficials { get; set; }
    }
}

File: client/src/app/_models/orders/orderForwardCategory.ts
import { IOrderForwardCategoryOfficial } from "./orderForwardCategoryOfficial";


export interface IOrderForwardCategory
{
	checked: boolean;
	id: number;
	orderId: number;
	orderNo: number;
	orderDate: Date;
	customerName: string;
	customerCity: string;

	orderItemId: number;
	professionId: number;
	professionName: string;
	charges: number;
	orderForwardCategoryOfficials: IOrderForwardCategoryOfficial[]
}

export class OrderForwardCategory implements IOrderForwardCategory
{
	checked= false;
	id= 0;
	orderId= 0;
	orderNo=0;
	orderDate=new Date();
	customerName='';
	customerCity='';
	
	orderItemId= 0;
	agentId= 0;
	professionId= 0;
	professionName= '';
	charges= 0;
	orderForwardCategoryOfficials: IOrderForwardCategoryOfficial[]=[];
}